storm-server/src/main/java/org/apache/storm/blobstore/KeySequenceNumber.java

```diff
@@ -131,9 +131,8 @@ public KeySequenceNumber(String key, NimbusInfo nimbusInfo) {
         this.nimbusInfo = nimbusInfo;
     }
 
-    public synchronized int getKeySequenceNumber(Map<String, Object> conf) throws KeyNotFoundException {
+    public synchronized int getKeySequenceNumber(CuratorFramework zkClient) throws KeyNotFoundException {
         TreeSet<Integer> sequenceNumbers = new TreeSet<Integer>();
-        CuratorFramework zkClient = BlobStoreUtils.createZKClient(conf);
```

Comment:
You should remove `finally` statement since it is no longer created in this method, it's being passed from caller. Caller should have responsibility to close this.

Reply:
done